### DISCUSSION TITLE
small farmers will be out

### DISCUSSION THREAD

**Discussion by mahdipsycho** [Chia-Network/chia-blockchain#6456] [General]
hey everyone
As I read in the main site and as I know 
I should calculate my profit like this
how much of the whole network is yours
we say 0.002% for example
and we have this
4608*2*0.002%
so 
0.002% of now is something about 360TB 
and If i dont want to reinvest or make my farm bigger
the network will rise and my percentage from 0.002% might go to 0.0002%
let's think about the day that we have a network of 300EiB 
everyone knows a day will come that network would be like this
and if someone wants to start from the begining with 100 TB of hdd storage( think we dont need even plotting)
so it would be something around 0.0000317891% of all of 300Eib network 
so he will get 0.0029296835 xch per day 
and if we think that chia is 700$ it would be something like 2$ per day
and we should invest something like 3120$ on hdds
and about 1500$ on plotter
so what will happen?
does it worth?
60$ per month with 4620$ investment
ROI would be 77 months if network stop rising on 300Eib and everyone knows it would not be possible 
so whats the answer?
chia would be 7000$?
if it be it worth to farm
share your opinion with me plz
this is important

**Comment by rafi7777** [comment]
I also believe that this Proof of Time based currency does not make sense for small players. The difficulty increases faster than my computer can create new threads. Sometimes it seems to me that CHIA was ordered by hard drive manufacturers.

**Comment by ktkiebel** [comment]
Chia hasnt done anything to promote decentralization as your comments also suggest.  The only obvious solution is to reduce the farming reward based on farm size.  It should be smaller anyway because larger operations are more efficient.  There's also a pool in china about half the size of the total network.  When that thing goes online it will be game over for individuals.
